src/index.ts

```diff
@@ -19,6 +20,18 @@ debug("Launching Impftermin");
 
 (async () => {
   const configuration = await loadConfiguration();
+  if (configuration.enableAutobooking == "true") {
+	enableAutobooking = true;
+  }else{
+	  enableAutobooking = false;
+	  debug(" ");
+	  debug("=============================================");
+	  debug("=============================================");
+      debug("= ATTENTION! Booking will only be simulated =");
```

Comment:
Please change this to something more user friendly like: `Automatic booking is not enabled. Enable it by adding the enableAutobooking: true to you config.json` . So they user knows what's happening.